include/zephyr/logging/log_msg2.h

```diff
@@ -276,6 +274,43 @@ do { \
 		(NULL), \
 		(Z_LOG_FMT_ARGS_2(_name, ##__VA_ARGS__)))
 
+#if defined(CONFIG_LOG2_USE_TAGGED_ARGUMENTS)
+
+#define Z_LOG_FMT_TAGGED_ARGS_2(_name, ...) \
+	COND_CODE_1(CONFIG_LOG2_FMT_SECTION, \
+		(COND_CODE_0(NUM_VA_ARGS_LESS_1(__VA_ARGS__), \
+		   (_name, Z_CBPRINTF_TAGGED_EMPTY_ARGS()), \
```

Comment:
Ideally, there should be a public cbprintf API for tagging something like: `CBPRINTF_TAGGED(...)`. It would handle string without arguments. Meanwhile, handling of string from format section would be kept in logging. Now it is mixed.

Reply:
Modified the macros so that it is using `Z_CBPRINTF_TAGGED_ARGS(...)` all the time.